### ISSUE TITLE
syntax error kiss lua on taranis qx7

### ISSUE THREAD

**Issue by fpvalert** [fedorcomander/kiss-gui#8]
hello guys. 

im trying to get my lua script to work but everything i try i get : script suntax error attempt to call a nil value.

i have flashet the newest open tx on my qx7 and got it set up to deal with lua and luac.
i hope you guys can help me out. 

slangejeff@hotmail.com

**Comment by damnfoile** [comment]
Hey, did you ever find a solution for this problem? 
I have the same issue with my taranis q x7 with open tx 2.3.10 and 
FrskyTelemetryScript-Taranis-1.8.0 or FrskyTelemetryScript-Taranis-1.9.1_beta1
both give the same error.

**Comment by fpvalert** [comment]
Yes. For me it was the wrong lua file or the way how i opened it.

If you just put it as telem screen it wont work for me, have to execute it thrue sd cart contents,


________________________________

**Comment by damnfoile** [comment]
Hey thanks a lot for the quick response. I tried to open it trough sd card, didn't work, guess i have the wrong file then.

**Comment by fpvalert** [comment]
If you dont come up with the fix, go ask on the kiss owners facebook page. People there will help you

Goodluck mate!
________________________________

**Comment by damnfoile** [comment]
Hey thanks a lot, turns out the files were placed in the wrong path,  I already make the script work, though the script now tells me "no telemetry".  I'm looking for solution to this now.
